packages/astro/src/assets/fonts/vite-plugin-fonts.ts

```diff
@@ -224,15 +227,39 @@ export function fontsPlugin({ settings, sync, logger }: Options): Plugin {
 				for (const data of fonts) {
 					css += generateFontFace(family.name, data);
 				}
+				const urls = fonts
+					.map((font) => font.src.map((src) => ('originalURL' in src ? src.originalURL : null)))
+					.flat()
+					.filter((url) => typeof url === 'string');
```

Comment:
```suggestion
					.filter(Boolean);
```

I think this works too